src/helm/benchmark/metrics/classification_metrics.py

```diff
@@ -43,23 +44,19 @@ def evaluate_instances(self, request_states: List[RequestState]) -> List[Stat]:
             if len(request_state.result.completions) != 1:
                 raise ValueError("Result must contain exactly one completion")
 
-            num_correct = 0
-            for reference in request_state.instance.references:
-                if reference.is_correct:
-                    num_correct += 1
-                    y_true.append(normalize_text(reference.output.text))
-            if num_correct != 1:
-                # TODO: Support multi-label classification.
-                raise ValueError("ClassificationMetric does not support multi-label classification")
+            references = request_state.instance.all_correct_references
+            correct_ref_texts = [ref.output.text.strip() for ref in references if ref.output.text]
+            y_true.append(correct_ref_texts)
+
             if request_state.output_mapping:
                 raise ValueError("ClassificationMetric does not support multiple choice adapters")
-            y_pred.append(normalize_text(request_state.result.completions[0].text))
-        labels = list(set(y_true))
+            predictions = request_state.result.completions[0].text.split(self.delimiter)
```

Comment:
In the single-label classification case, you should not split the label at all, because the single label could contain the delimiter! This is why the `__init__` above needs to know if this is single-label or multi-label.